core/encoding/json/marshal.odin

```diff
@@ -488,38 +535,75 @@ marshal_to_writer :: proc(w: io.Writer, v: any, opt: ^Marshal_Options) -> (err:
 			}
 			return false
 		}
+		
+		bit_data: u128
+		bit_size := u128(8*ti.size)
 
-		bit_data: u64
-		bit_size := u64(8*ti.size)
-
-		do_byte_swap := is_bit_set_different_endian_to_platform(info.underlying)
+		do_byte_swap := is_bit_set_different_endian_to_platform(ti)
```

Comment:
`info.underlying` -> `ti` seems like a mistake? `ti` is going to be the bit set right? and the `is_bit_set_different...` is checking/expecting an integer.

afaict this is now always going to return false.

Reply:
that would make sense, I'll look into it and fix it. Thank you for the review